extensions/parquet/table/src/main/java/io/deephaven/parquet/table/pagestore/topage/ToPage.java

```diff
@@ -105,13 +106,18 @@ default ToPage<DictionaryKeys, long[]> getDictionaryKeysToPage() {
     }
 
     /**
-     * @return an reverse lookup map of the dictionary.
+     * @return a reverse lookup map of the dictionary.
      * @apiNote null iff {@link #getDictionaryChunk()} is null.
      */
     default LongBitmapStringSet.ReversibleLookup getReversibleLookup() {
         return null;
     }
 
+    /**
+     * @return the factory to create the materializers for this column.
+     */
+    PageMaterializerFactory getPageMaterializerFactory();
```

Comment:
Would it be cleaner if instead of having a `PageMaterializerFactory`, `ToPages` implemented `PageMaterializerFactory` themselves? That seems more aligned with the intent, here.